components/mediation/inbound-endpoints/org.wso2.micro.integrator.inbound.endpoint/src/main/java/org/wso2/micro/integrator/inbound/endpoint/protocol/grpc/util/EventOrBuilder.java

```diff
@@ -0,0 +1,53 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Shall we add the license header please

Reply:
@Nirothipan fixed in https://github.com/wso2/micro-integrator/pull/772/commits/06f7426b0a48c85a38c3c29ba4cd3c52dccea7e5